src/api/keywordsAPI.ts

```diff
@@ -1,14 +1,10 @@
-import { authApiGet } from "./apiUtils";
-
-export interface Keyword {
-  keywordId: number;
-  keywordName: string;
-}
-
-type GetKeywordsResponse = {
-  [key: string]: Keyword[];
-};
+import { authApiGet, authApiPut } from "./apiUtils";
+import { GetKeywordsResponse, putKeywordsData } from "./types/keywords";
 
 export const getKeywords = async (clientId: number) => {
   return authApiGet<GetKeywordsResponse>("/keywords", { clientId });
 };
+
+export const putKeywords = async (clientId: number, data: putKeywordsData) => {
```

Comment:
clientId 없어질거라서 안해도 되긴 하는데 원래 clientId 없어지는거 아니면
```suggestion
interface putKeywordsParams {
clientId: number
data: putKeywordsData
}
export const putKeywords = async ({clientId, data}: putKeywordsParams) =>
```
이렇게 해야 합니다